Source/player.h

```diff
@@ -151,177 +151,177 @@ struct PlayerStruct {
 	int8_t walkpath[MAX_PATH_LENGTH];
 	bool plractive;
 	action_id destAction;
-	Sint32 destParam1;
-	Sint32 destParam2;
+	int destParam1;

```

Comment:
Use 'int32_t' in structures used for serialization, to express that the width here matters a lot?

Reply:
Only cmd_*, PKPlayerStruct and PKItemStruct are serialized.

Reply:
Ah, I didn't mean 'directly serialized'. I meant that in the sense of "There's code somewhere that reads/writes this value with the expectation of it being a certain size".

I haven't tried, but what happens if you e.g. use writeLE<Uint8>(playerStruct.destParam1)? Will it complain about the narrowing conversion?